conjure-http/src/server/mod.rs

```diff
@@ -36,6 +35,10 @@ use std::str;
 use std::str::FromStr;
 use std::sync::Arc;
 
+use crate::private::{self, APPLICATION_JSON, SERIALIZABLE_REQUEST_SIZE_LIMIT};
+
+pub mod conjure;
```

Comment:
conjure-codegen will use types from both the top level module and the conjure submodule, but I figured it'd be best to split off the types that handle weird conjure-specific cases like `optional<binary>` and the `plain` encoding.